accel/opencl/matmul.cl

```diff
@@ -18,8 +18,10 @@ __kernel void matmul(
   // fast path precompute (32x speedup)
   float outputValue = 0.0f;
   for (short inputSet = (short)get_global_id(1); inputSet < numPackedInputChannelsForGroup; inputSet += get_global_size(1)) {
-    float4 inputValues = read_imagef(input, smp, inputSet);
-    float4 weightValues = read_imagef(weights, smp, (int2)(mad24(inputSet, 4, weightIndex), packedOutputChannel));
+    int2 inputLocation = (int2)(inputSet, 0);
+    float4 inputValues = read_imagef(input, smp, INPUT_LOCATION.x);
```

Comment:
Why is there .x here?

Reply:
This is a `read_imagef` (1D) call, so passing an int2 sampling location wouldn't compile. That said, basic 1x1 conv tests with `OPENCL=1 MATMUL=1` seem to fail both after and be3fore this PR, so I have little confidence I've edited it correctly.

Reply:
Yea, MATMUL is broken on PC for some reason, this might need to be fixed first...let me look

Reply:
Thanks!